### PR TITLE
Shyft network

### LINKED ISSUE #849: new chain : shyft
Hi, I tried to make a branch and commit to make a pull request and make this easy but this is not alllowed.

Here are the mainnet and testnet definitions for http://shyft.network/
I have placed the icon in IPFS.

icons/shyft.json:
```
[
    {
	"url":"ipfs:QmUkFZC2ZmoYPTKf7AHdjwRPZoV2h1MCuHaGM4iu8SNFpi",
	"width":400,
	"height":400,
	"format":"svg"
    }
]
```

chains/eip155-7341.json:
```
{
  "name": "Shyft Mainnet",
  "chain": "SHYFT",
  "icon": "shyft",
  "rpc": [
    "https://rpc.shyft.network/",
  ],
  "faucets": [],
  "nativeCurrency": {
    "name": "Shyft",
    "symbol": "SHYFT",
    "decimals": 18
  },
  "infoURL": "https://shyft.network",
  "shortName": "shyft",
  "chainId": 7341,
  "networkId": 7341,
  "slip44": 2147490989,
  "explorers": [{
    "name": "Shyft BX",
    "url": "https://bx.shyft.network",
    "standard": "EIP3091"
  }]
}
```

chains/eip155-11437.json:
```
{
  "name": "Shyft Testnet",
  "chain": "SHYFTT",
  "icon": "shyft",
  "rpc": [
    "https://rpc.testnet.shyft.network/",
  ],
  "faucets": [],
  "nativeCurrency": {
    "name": "Shyft Test Token",
    "symbol": "SHYFTT",
    "decimals": 18
  },
  "infoURL": "https://shyft.network",
  "shortName": "shyftt",
  "chainId": 11437,
  "networkId": 11437,
  "explorers": [{
    "name": "Shyft Testnet BX",
    "url": "https://bx.testnet.shyft.network",
    "standard": "EIP3091"
  }]
}
```

### PR DESCRIPTION
**Pull request by svanegmond**
closes #849

### CODE REVIEW

File: _data/chains/eip155-11437.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shyft Testnet",
+  "chain": "SHYFTT",
+  "icon": "shyft",
+  "rpc": [
+    "https://rpc.testnet.shyft.network/",
```
Comment (ligi) [inline _data/chains/eip155-11437.json:6]:
```suggestion
    "https://rpc.testnet.shyft.network/"
```

File: _data/chains/eip155-7341.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Shyft Mainnet",
+  "chain": "SHYFT",
+  "icon": "shyft",
+  "rpc": [
+    "https://rpc.shyft.network/",
```
Comment (ligi) [inline _data/chains/eip155-7341.json:6]:
```suggestion
    "https://rpc.shyft.network/"
```

**Review by ligi — CHANGES_REQUESTED** [review]
JSON is not valid - please apply requested changes

File: _data/chains/eip155-11437.json (hunk 1 above)
Comment (svanegmond) [reply _data/chains/eip155-11437.json:6]: Apologies, thank you.

Comment (ligi) [reply _data/chains/eip155-11437.json:6]: you need to commit this change

**Comment by svanegmond** [comment]
I was unaware the same problem existed in the other file I patched. Committed a patch for that too.

**Pull request merged**